com.unity.ml-agents/Runtime/Academy.cs

```diff
@@ -32,7 +32,14 @@ internal class AcademyFixedUpdateStepper : MonoBehaviour
     {
         void FixedUpdate()
         {
-            Academy.Instance.EnvironmentStep();
+            if (!Academy.IsStepperOwner(this))
```

Comment:
I think you should be a little more careful here with how you access the Academy instance so that we don't create a new one accidentally. I'd recommend:

1) Make IsStepperOwner() non-static again
2) In FixedUpdate(), either do nothing if the Academy isn't initialized, or call Destroy() (but avoid the call to IsStepperOwner).